CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Contributing to Headset
+
+Welcome to the Headset community!
+
+## Community
+
+We have two main channels of communication:
+- our Slack workspace: https://tinyurl.com/y7m8y5x4
+- issues on GitHub
+
+If you have any questions on where to start and/or how to proceed, let us know!
+
+## Ways to contribute
+
+We welcome many ways of contributing to Headset:
+- finding bugs and [reporting issues](https://github.com/headsetapp/headset-electron/issues)
+- picking and working on an existing [issue](https://github.com/headsetapp/headset-electron/labels/help-wanted) marked as 
+  "help wanted"
+- proposing and implementing improvements to Headset
+
+## Setting up the environment
+
+1. Fork the repository on GitHub
+
+2. Clone your fork of the repository
+
+  ```
+  git clone https://github.com/<your-username>/headset-electron.git
+  ```
+
+3. Installation is OS-dependent, so choose the appropriate OS directory:
+  - `windows` for Windows
+  - `darwin` for OSX
+  - `linux` for Linux
+
+    To install all the required dependencies run:
+
+    ```
+    npm i && cd <your-os> && npm i
```

Comment:
We should change `<your-os>` to just `<os>` because people can develop for multiple OSs, especially if their system is OSX or Linux. The tests are OS dependent though.

Also, more important, this command will fail on Windows (sorry, I develop on Linux and tend to forget Windows). The command should be `npm install; cd <os>; npm install`. This will work on all OSs (using PowerShell for Windows)